tests/test_utils.py

```diff
@@ -20,3 +21,9 @@ class TestCase(unittest.TestCase):
         [torch.zeros(1, 3, 3, 3), torch.ones(1, 3, 3, 3), torch.full((1, 3, 3, 3), 2)],
         1,
     )
```

Comment:
TestCase -> TestCaseConfig

Reply:
`TestCase` was removed and config, image etc. are now imported directly from `test_utils`